modules/core/location/tests/src/Kernel/LocationTest.php

```diff
@@ -318,6 +318,16 @@ public function testAssetLocation() {
     // Assert that the asset's cache tags were not invalidated.
     $this->assertEntityTestCache($asset, TRUE);
 
+    // Change the log status to "abandoned" and confirm that the asset location
+    // is unchanged.
+    $second_log->set('status', 'abandoned');
+    $second_log->save();
+    $this->assertEquals($this->logLocation->getLocation($first_log), $this->assetLocation->getLocation($asset), 'Asset with abandoned movement log has original location');
+    $this->assertEquals($this->logLocation->getGeometry($first_log), $this->assetLocation->getGeometry($asset), 'Asset with abandoned movement log has original geometry.');
+
+    // Assert that the asset's cache tags were not invalidated.
+    $this->assertEntityTestCache($asset, TRUE);
+
```

Comment:
Ditto. Just depends how involved we want to get with tests...

Reply:
Agreed. Will do.